test/e2e/main_test.go

```diff
@@ -69,10 +69,23 @@ var _ = BeforeEach(func() {
 	bridge1 = nextBridge()
 	_, namespace = prepare(t)
 	startTime = time.Now()
+	println("Getting nodes initial state")
```

Comment:
Please use `By` from the testing framework. The same for all the other `println`

Reply:
Sorry :man_facepalming: